### PR TITLE
Viceroy API test

### PR DESCRIPTION
**Pull request by h-parekh**
This test only checks for a 200 response code and the scenario when a user is not logged in.

### CODE REVIEW

**Comment by h-parekh** [comment]
@jeremyf: ping

File: spec/viceroy/integration/int_viceroy_spec.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 require 'viceroy/viceroy_spec_helper'
 
-describe 'API tests using swagger definition' do
+describe 'Viceroy API test' do
   SwaggerHandler.operations(for_file_path: __FILE__).each do |operation|
     it "calls #{operation.verb} #{operation.path}" do
       result = public_send(operation.verb, operation.url)
       expect(result.code.to_s).to eq(operation.responses.keys[0])
-      expect_json_types(message: :string)
+      expect_json_types(redirect: :string)
```
Comment (h-parekh) [inline spec/viceroy/integration/int_viceroy_spec.rb]: The actual response from the API endpoint is: {"redirect":"https://viceroy-dev.library.nd.edu/login"}

I'm asserting for the presence of 'redirect' field.

Comment (h-parekh) [inline spec/viceroy/integration/int_viceroy_spec.rb]: @jeremyf - Did I answer your question?

Comment (jeremyf) [inline spec/viceroy/integration/int_viceroy_spec.rb]: Yes it did.

**Pull request merged**